### ISSUE TITLE
Formbuilder client feedback

### ISSUE THREAD

**Issue by toddwseattle** [eecs394-s16/PitchEvaluator#13]
Consider a more traditional forms design with the team on the left and right; though maybe what you have will look better on mobile.
For forms, if you are using bootstrap, consider using: [bootstrap forms builder](http://bootsnipp.com/forms) to make it easier to design

**Comment by BaiyuY** [comment]
What do you mean by "team on the left and right"? Do you mean we have a list of team on the right or left of the screen like tabs, then display the form on the other side?

**Comment by toddwseattle** [comment]
no, i was just refering to the labels on the forms.  sorry; not really team; but the team label.
